client/src/components/Home/SectionOne.jsx

```diff
@@ -1,15 +1,18 @@
 // import { Element } from "react-scroll";
 
-import { Link } from "react-router-dom";
-import gitbook from "./svgs/GitBookLight.svg";
-import msme from "./svgs/MSME.svg";
-import microsoft from "./svgs/microsoft4strp.svg";
-import digitalOcean from "./svgs/digitalOcean.svg";
 import meltcd from "@assets/meltcd.png";
+import { useUserAuth } from "@context/UserAuthContext";
 import { Star } from "@mui/icons-material";
 import { motion } from "framer-motion";
+import { GoArrowUpRight } from "react-icons/go";
 import { MdPlayArrow } from "react-icons/md";
+import { Link } from "react-router-dom";
+import gitbook from "./svgs/GitBookLight.svg";
+import msme from "./svgs/MSME.svg";
+import digitalOcean from "./svgs/digitalOcean.svg";
+import microsoft from "./svgs/microsoft4strp.svg";
```

Comment:
any reason why you changed the positioning of these import statements?

Reply:
> any reason why you changed the positioning of these import statements?

its prettier auto format on save. nothing more :)